core/src/components/picker-column/picker-column.tsx

```diff
@@ -465,22 +478,161 @@ export class PickerColumn implements ComponentInterface {
     this.el.classList.remove('picker-column-active');
   };
 
-  get activeItem() {
+  get activeItem(): {
+    activeItem?: HTMLIonPickerColumnOptionElement;
+    activeItemIndex: number;
+  } {
```

Comment:
This refactor isn't 100% necessary to ship this fix, so let me know if you'd like me to split this out. In order to correctly set `aria-valuenow` I need to now where the active options appears in a list of all options (its index) which requires an iteration. Rather that do 2 iterations, I decided to update `activeItem` to return all the information I need from a single iteration.